src/components/Item/VisualizationItem/Visualization/__tests__/Visualization.spec.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import configureMockStore from 'redux-mock-store'
+
+import Visualization from '../Visualization'
+
+jest.mock('@dhis2/data-visualizer-plugin', () => 'VisualizationPlugin')
+jest.mock('../MapPlugin', () => 'MapPlugin')
+jest.mock('../DefaultPlugin', () => 'DefaultPlugin')
+
+const mockStore = configureMockStore()
+
+test('renders a MapPlugin when activeType is MAP', () => {
+    const store = {
+        visualizations: { rainbowVis: { rows: [], columns: [], filters: [] } },
```

Comment:
`visualizations: { rainbowVis: { rows: [], columns: [], filters: [] } },` is repeated 5 times. Set it as e.g. `const DEFAULT_STORE_WITH_1_ITEM = ...` and use `const store = DEFAULT_STORE_WITH_1_ITEM` inside the tests?

Reply:
Done.